provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -5,7 +5,7 @@
 set -eu
 set -o pipefail
 
-CORIM_TEMPLATE=corimMini.json
+CORIM_CCA_TEMPLATE=corimMini.json
```

Comment:
as a general comment, this has become a tad chaotic.  please consider organising the flow a bit more linearly.